app/auth/reducer.spec.js

```diff
@@ -26,6 +30,7 @@ describe('auth/reducer', () => {
       it('sets state to payload.auth', () => {
         const payload = {
           auth: {
+            isFetched: true,
```

Comment:
- [ ] isFetched should not be in the payload. But you should test that it is actually set to true with AUTH_GET_SUCCESS action.